src/app/util/attribute-storage.cpp

```diff
@@ -131,15 +133,34 @@ void emberAfEndpointConfigure(void)
     uint8_t fixedNetworks[]             = FIXED_NETWORKS;
 #endif
 
-    emberEndpointCount = FIXED_ENDPOINT_COUNT;
+#if ZAP_FIXED_ENDPOINT_DATA_VERSION_COUNT > 0
+    // Initialize our data version storage.  If
+    // ZAP_FIXED_ENDPOINT_DATA_VERSION_COUNT == 0, gcc complains about a memset
+    // with size equal to number of elements without multiplication by element
+    // size, because the sizeof() is also 0 in that case...
+    if (Crypto::DRBG_get_bytes(reinterpret_cast<uint8_t *>(fixedEndpointDataVersions), sizeof(fixedEndpointDataVersions)) !=
```

Comment:
Why random?

Reply:
DataVersion is tested for equality on the server.

So what we want is to end up with a different value than the one the client had, if the data version changes.  The options are either to do something deterministic, where we save it to persistent storage when it changes and read it on boot, or to just init it to a random value on boot and assume that the chance of collision with a value a client is hoding is very low...  This is implementing the latter, per @mrjerryjohns request.

Reply:
From the spec:

> A cluster data version SHALL be initialized randomly when it is first published